specs/MultiProfile.md

```diff
@@ -217,6 +232,76 @@ HRESULT AppWindow::OnCreateCoreWebView2ControllerCompleted(
 }
 ```
 
+### Manage password-autosave and general-autofill settings in profile
+
+```cpp
+HRESULT AppWindow::ManagePasswordAutosaveInProfile(ICoreWebView2Controller* controller)
+{
+    // ...
+
+    // Get the profile object.
+    wil::com_ptr<ICoreWebView2> coreWebView2;
+    CHECK_FAILURE(m_controller->get_CoreWebView2(&coreWebView2));
+    Microsoft::WRL::ComPtr<ICoreWebView2Profile> webView2Profile;
+    CHECK_FAILURE(webView2->get_Profile(&webView2Profile));
```

Comment:
If the `CoreWebView2Profile.IsPasswordAutosaveEnabled` property is synchronized to `CoreWebview2Settings.IsPasswordAutosaveEnabled`, then why do we even need to go to the profile? Couldn't this sample be simplified to

```cpp
    wil::com_ptr<ICoreWebView2> coreWebView2;
    CHECK_FAILURE(m_controller->get_CoreWebView2(&coreWebView2));
    Microsoft::WRL::ComPtr<ICoreWebView2Settings> webView2Settings; // Use the existing Settings properties
    CHECK_FAILURE(webView2->get_Settings(&webView2Settings));
    Microsoft::WRL::ComPtr<ICoreWebView2Settings4> webView2Settings4;
    if (SUCCEEDED(webView2Settings.As(&webView2Settings4)))
    {
        BOOL wasEnabled;
        CHECK_FAILURE(webView2Settings4->get_IsPasswordAutosaveEnabled(&wasEnabled));
        CHECK_FAILURE(webView2Settings4->put_IsPasswordAutosaveEnabled(!wasEnabled));
        MessageBox(MainWindow(), wasEnabled ?
            L"Password autosave will be disabled at next navigation" :
            L"Password autosave will be enabled at next navigation", MB_OK);
    }
}
```

Reply:
Yes, using `CoreWebview2Settings` is the same as `CoreWebView2Profile`, but I personally prefer using `CoreWebView2Profile` in the sample due to two reasons:
1. `IsPasswordAutosaveEnabled` and `IsGeneralAutofillEnabled` are actually profile-level properties, while most properties in `CoreWebview2Settings` could only take effect on a single WebView.
2. It seems be better if we can provide some samples to get/set these two properties through profile object.

Reply:
Please update the documentation for the existing properties to explain that duplication.

Reply:
If I understand correctly, you mean we should update the documentation for `CoreWebView2Settings. IsPasswordAutosaveEnabled` and `CoreWebView2Settings.IsGeneralAutofillEnabled`? If so, we have updated it in the ADO repo, is there any other documentation that I need to update?

Reply:
There's also the public doc page
* [CoreWebView2Settings.IsGeneralAutofillEnabled](https://learn.microsoft.com/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/CoreWebView2Settings#isgeneralautofillenabled)
* [CoreWebView2Settings.IsPasswordAutosaveEnabled](https://learn.microsoft.com/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/CoreWebView2Settings#ispasswordautosaveenabled)

Reply:
> It seems be better if we can provide some samples to get/set these two properties through profile object.

But the functionality of this sample doesn't require the new API. You can do the same thing with the old API. The sample doesn't really show why the new API is needed.

Maybe the point is "The new API isn't really needed, but it's less confusing."

Or maybe the sample could operate directly on a profile (not associated with a WebView controller.)

Reply:
> Maybe the point is "The new API isn't really needed, but it's less confusing."

Yes it is this.